pkg/scheduler/extender_test.go

```diff
@@ -134,6 +142,7 @@ func TestGenericSchedulerWithExtenders(t *testing.T) {
 			},
 			extenders: []st.FakeExtender{
 				{
+					ExtenderName: "FakeExtenderName1",
```

Comment:
if we have default ExtenderName, do we need this?

Reply:
Of course, it will work without it, but I think it's better to have it. 
(For example, if someone wants to add new test case with reference to existing test cases, he/she can notice that it (may) need ExtenderName.)